crates/integration_test/src/utils/support.rs

```diff
@@ -78,7 +78,7 @@ pub fn prepare_bundle(id: &Uuid) -> Result<TempDir> {
 
     let mut spec = Spec::default();
     let mut process = Process::default();
-    process.set_args(Some(vec!["sleep".into(), "5".into()]));
+    process.set_args(Some(vec!["sleep".into(), "10".into()]));
```

Comment:
Was this changed for any specific reason? There is no issue with changing this, as the original value 5 was also arbitrary, but if there is a specific reason, can you either put a comment here or extract the &'static str as a const and name it appropriately?

Reply:
Ah sorry, I was going to highlight it here. The reason for bumping it to 10 was that lifecycle tests were failing on my machine (before any changes). The process was exiting before the test managed to kill it, so kill was failing on the stopped container.

I suppose this does not happen on CI so I can revert this change, however, it might be useful for other people running on slower machines.

Reply:
I guess that's fine, as I said there was no specific reason for keeping it 5 either ...